### PR TITLE
Enhance signal templates

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**
- Rename `Send_Signal_To_Job` to `Send_Signal_To_Another_Job`
- Add `Send_Signal_With_Variables_To_Another_Job`

### CODE REVIEW

File: Controls/resources/catalog/Send_Signal_With_Variables_To_Another_Job.xml [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+  <variables>
+    <variable name="IDENTIFY_JOB" value="Use_Job_ID" model="PA:LIST(Use_Job_ID, Use_Workflow_Name_and_Variable)" description="How to identifiy the job to which the signal will be sent." group="Job Identification Parameters" advanced="false" hidden="false"/>
+    <variable name="JOB_ID" value="0"  description="ID of the job to which the signal will be sent." group="Job Identification Parameters" advanced="false" hidden="false"/>
+    <variable name="WORKFLOW_NAME" value="controls/Wait_For_Signals_Reactive"  description="Name of the workflow of the job to which the signal will be sent." group="Job Identification Parameters" advanced="false" hidden="false"/>
+    <variable name="WORKFLOW_VARIABLE_KEY" value=""  description="Name of a variable of the job to which the signal will be sent." group="Job Identification Parameters" advanced="false" hidden="false"/>
```
Comment (fviale) [inline Controls/resources/catalog/Send_Signal_With_Variables_To_Another_Job.xml:9]: I don't understand well this paramer. Why not use a Json definition of variables instead ?

File: Controls/resources/catalog/Send_Signal_With_Variables_To_Another_Job.xml [hunk 2]
```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="Send_Signal_With_Variables_To_Another_Job" projectName="4. Signal Templates" tags="Controls,Get started,Building blocks" priority="normal" onTaskError="cancelJob"  maxNumberOfExecution="1"  >
+  <variables>
+    <variable name="IDENTIFY_JOB" value="Use_Job_ID" model="PA:LIST(Use_Job_ID, Use_Workflow_Name_and_Variable)" description="How to identifiy the job to which the signal will be sent." group="Job Identification Parameters" advanced="false" hidden="false"/>
+    <variable name="JOB_ID" value="0"  description="ID of the job to which the signal will be sent." group="Job Identification Parameters" advanced="false" hidden="false"/>
+    <variable name="WORKFLOW_NAME" value="controls/Wait_For_Signals_Reactive"  description="Name of the workflow of the job to which the signal will be sent." group="Job Identification Parameters" advanced="false" hidden="false"/>
```
Comment (fviale) [inline Controls/resources/catalog/Send_Signal_With_Variables_To_Another_Job.xml:8]: why having both JOB_ID and WORKFLOW_NAME ?

**Review by AlexFalessi — APPROVED** [review]
Why have two worfklows ? Can't have one where sending variables is optional ?

File: Controls/resources/catalog/Send_Signal_With_Variables_To_Another_Job.xml (hunk 1 above)
Comment (nebilbenmabrouk) [reply Controls/resources/catalog/Send_Signal_With_Variables_To_Another_Job.xml:9]: This workflow is devised to search for the target job (to which the signal will be sent) based on either:
- JOB_ID
- WORKFLOW_NAME + WORKFLOW_VARIABLE_KEY + WORKFLOW_VARIABLE_VALUE

File: Controls/resources/catalog/Send_Signal_With_Variables_To_Another_Job.xml (hunk 2 above)
Comment (nebilbenmabrouk) [reply Controls/resources/catalog/Send_Signal_With_Variables_To_Another_Job.xml:8]: See the above explanation

Comment (fviale) [reply Controls/resources/catalog/Send_Signal_With_Variables_To_Another_Job.xml]: ok

**Review by fviale — APPROVED** [review]

**Pull request merged**